src/frontend/ocamlmerlin/new/new_commands.ml

```diff
@@ -564,8 +564,11 @@ The return value has the shape:
     begin fun pipeline () ->
       let config = Mpipeline.final_config pipeline in
       `Assoc [
+        (* TODO: print the configuration context. *)
```

Comment:
@voodoos TODO
@trefis  Maybe you can clarify what is expected here ? Just printing the .merlin content ?

Reply:
Path vers .merlin + lister les erreurs